streamer/src/nonblocking/testing_utilities.rs

```diff
@@ -141,28 +142,20 @@ pub fn setup_quic_server(
     let sockets = {
         #[cfg(not(target_os = "windows"))]
         {
-            use std::{
-                os::fd::{FromRawFd, IntoRawFd},
-                str::FromStr as _,
-            };
             (0..10)
                 .map(|_| {
-                    let sock = socket2::Socket::new(
-                        socket2::Domain::IPV4,
-                        socket2::Type::DGRAM,
-                        Some(socket2::Protocol::UDP),
+                    bind_to(
+                        IpAddr::V4(Ipv4Addr::LOCALHOST),
+                        /*port*/ 0,
+                        /*reuseport:*/ true,
```

Comment:
Just making sure - we couldn't use the convenience helper since `reuseport = true` here ?

Reply:
correct, although we could change the helper to always accept a reuse port argument. or add a separate helper that is something like: `bind_to_localhost_reuse_port()` but that feels excessive.